### PR TITLE
style : Change alert text margin

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
change alert text margin and alert-info font-color

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
You can merge as code looks clean.

File: Components/alert.css [hunk 1]
```diff
@@ -43,7 +42,7 @@
   }
   
   .alert-info {
-    color: var(--info);
+    color: var(--blue);
```
Comment (Samarth-5109) [inline Components/alert.css:45]: avoid naming variables like blue, red, etc.

**Review by Samarth-5109 — APPROVED** [review]
looks fine added a comment

File: Components/alert.css (hunk 1 above)
Comment (Omkar-Chaskar) [reply Components/alert.css:45]: ok..

**Pull request merged**